newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/index.js

```diff
@@ -161,8 +175,31 @@ export default class LinkEvent extends React.Component<EventRendererProps, *> {
                     }}
                     tabIndex={0}
                   >
-                    {target || (
-                      <Trans>{`<Enter the name of external events>`}</Trans>
+                    {this.isLinkedToWrongLayout(target) ? (
+                      <span
+                        className={classNames({
+                          [instructionWarningParameter]: true,
```

Comment:
Would be surely worth to create a new css class for this - this is not really a parameter and we can see the icon is too small compared to the text.

Reply:
I added a CSS rule for an icon in an event link. As the warning class only affect the color and underlining, I guess we could rename `instruction-warning-parameter` to something like `warning-parameter` one day.